plugins/com.google.cloud.tools.eclipse.appengine.libraries/plugin.xml

```diff
@@ -318,10 +318,10 @@
           name="%cloudtranslation.name"
           tooltip="%cloudtranslation.tooltip"
           javaVersion="1.8"
-          siteUri="https://cloud.google.com/translate/docs/reference/libraries" >
-      <libraryDependency id="googlecloudcore" />
+          siteUri="https://cloud.google.com/translate/docs/reference/libraries">
       <libraryFile
-            javadocUri="https://googlecloudplatform.github.io/google-cloud-java/0.21.0/apidocs/index.html?com/google/cloud/translate/package-summary.html">    
+          loadDependencies="true"
+          javadocUri="https://googlecloudplatform.github.io/google-cloud-java/0.21.0/apidocs/index.html?com/google/cloud/translate/package-summary.html">    
```

Comment:
Since we use this to feed into Eclipse JDT for javadoc hovers, this should be just `https://googlecloudplatform.github.io/google-cloud-java/0.21.0/apidocs`. This is only used for specific types and methods, rather than a general entry point to documentation.

Reply:
Do we use it for that though? I looked and didn't find the path where that happened. I'm not 100% sure we're using this at all.